### PR TITLE
More tests for coverage and some fixes

### PR DESCRIPTION
**Pull request by yusufsafak**
- fix for #816 #817 
- remove unused convertChangeMap function from DependencyStore
- combine some codes into setTrackMap in DependencyTracker
- fix invalid path error in Model updateIn, updateInFrozen method and remove doubled code
- remove deprecated compute code from Tag
- fix runSignal unserializable payload warning
- fix addModule and removeModule methods. Now they support also subModules

### CODE REVIEW

**Comment by Guria** [comment]
Why some tests is commented out?

File: packages/cerebral/src/providers/Recorder.test.js [hunk 1]
```diff
@@ -386,4 +386,90 @@ describe('Recorder', () => {
...
+    assert.deepEqual(controller.getState(), {
+      foo: 'bar2'
+    })
+    timeout.tick()
+    assert.deepEqual(controller.getState(), {
+      foo: 'bar3'
+    })
+    controller.getSignal('stop')()
+  })
+  /* it.only('should throw an error when sending play signal while already playing or recording', () => {
```
Comment (yusufsafak) [inline packages/cerebral/src/providers/Recorder.test.js]: it doesn't work :( I need help for this

File: packages/cerebral/src/Controller.test.js [hunk 2]
```diff
@@ -11,20 +11,20 @@ describe('Controller', () => {
...
-  /* it('should warn devtools recommendation', () => {
+  it('should warn devtools recommendation', () => {
     let warnCount = 0
     const originWarn = console.warn
     console.warn = function (...args) {
       warnCount++
       originWarn.apply(this, args)
     }
-    navigator = {userAgent: 'Chrome'}
+    navigator = {userAgent: 'Chrome'} // eslint-disable-line
```
Comment (yusufsafak) [inline packages/cerebral/src/Controller.test.js]: it gives lint error. I disabled it

Comment (henri-hulski) [inline packages/cerebral/src/Controller.test.js]: If it's a globals error you can add at the top of the file 
```js
/* globals userAgent */
```

**Comment by Guria** [comment]
cc @christianalfoni this PR requires your deep dive into it

File: packages/cerebral/src/providers/Recorder.test.js (hunk 1 above)
Comment (Guria) [inline packages/cerebral/src/providers/Recorder.test.js]: @christianalfoni what about this one?

**Comment by yusufsafak** [comment]
Thanks! :)

**Pull request merged**